pkg/kubectl/cmd/util/openapi/openapi.go

```diff
@@ -77,48 +79,53 @@ func (d *document) LookupResource(gvk schema.GroupVersionKind) proto.Schema {
 }
 
 // Get and parse GroupVersionKind from the extension. Returns empty if it doesn't have one.
-func parseGroupVersionKind(s proto.Schema) schema.GroupVersionKind {
+func parseGroupVersionKind(s proto.Schema) []schema.GroupVersionKind {
 	extensions := s.GetExtensions()
 
+	gvkListResult := []schema.GroupVersionKind{}
+
 	// Get the extensions
 	gvkExtension, ok := extensions[groupVersionKindExtensionKey]
 	if !ok {
-		return schema.GroupVersionKind{}
+		return gvkListResult
 	}
 
-	// gvk extension must be a list of 1 element.
+	// gvk extension must be a list of at least 1 element.
 	gvkList, ok := gvkExtension.([]interface{})
 	if !ok {
-		return schema.GroupVersionKind{}
+		return gvkListResult
 	}
-	if len(gvkList) != 1 {
-		return schema.GroupVersionKind{}
+	if len(gvkList) == 1 {
+		return gvkListResult
```

Comment:
this looks wrong.  The entire block should go

Reply:
Ah meant to change to `== 0`, but I agree, this block does not add anything now; will remove